docs/contributing/adding_a_new_resource.md

```diff
@@ -53,35 +53,51 @@ A few important things to note when adding functions that call the AWS API:
 
 ### Integration Tests
 
-To Run Integration tests please run
-
+To prepare your environment for running integration tests:
 ```bash
 docker run -p 5432:5432 -e POSTGRES_PASSWORD=pass -d  postgres:13.3
```

Comment:
Also - a question. Do we need to specify the `postgres` version for this to work?
CI uses latest https://github.com/cloudquery/cq-provider-aws/blob/8603791b3bbfbd924ade4271cb5472f98418d7a7/.github/workflows/test_integration.yml#L44 so maybe we can do the same here?

Reply:
mock tests run vs 11,12, 13. Latest is using version 14 which is okay. We support >11. I don't think its useful to run e2e tests on all versions, mock tests is enough and usually faster.